nvtabular/workflow.py

```diff
@@ -89,7 +86,6 @@ def __init__(
 
         self.stats = {}
         self.ds_exports = export_path
```

Comment:
Doesn't seem like the workflow is using `self.ds_exports` either?

Reply:
Good catch - it was only being used in the unittests. removed in the latest commit